ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -19,9 +19,9 @@
 
 @dataclasses.dataclass
 class TtsTask:
+    template: str = simple_parsing.field(alias="-T")
```

Comment:
Let's leave this where it was and just add `default=None` like the other similar fields

Reply:
Does it make sense for template to be `None`?

Reply:
default=None doesnt pass the checks. I just moved it in the front.

Reply:
ah yeah, this is non-optional. LG.